### PR TITLE
Add a road network gazetteer

### LINKED ISSUE #44: Create a gazetteer for road networks
To help users QA their style's road network, we should update the gazetteer so users can preview their bridges, roads, and tunnels and multiple zoom levels and locations.

### PR DESCRIPTION
**Pull request by ClareTrainor**
Created a road network gazetteer! This includes 14 places from zoom levels 6-18. Below are the screenshots from cartocam.

@dasulit this is ready for review. Let me know if you have any feedback on the number of locations I have chosen/the range of places. 

Closes #44

Screen Shot 2019-07-02 at 9 35 47 PM
Screen Shot 2019-07-02 at 9 35 30 PM
Screen Shot 2019-07-02 at 9 35 26 PM
Screen Shot 2019-07-02 at 9 35 18 PM
Screen Shot 2019-07-02 at 9 35 12 PM
Screen Shot 2019-07-02 at 9 35 02 PM
Screen Shot 2019-07-02 at 9 34 55 PM

### CODE REVIEW

**Comment by dasulit** [comment]
@ClareTrainor if you rebase to master, tests should pass!

**Comment by ClareTrainor** [comment]
@dasulit added some descriptions, rebased, and all tests have passed! Let me know if you see anything else.

File: mapbox-streets/road-network-gazetteer.json [hunk 1]
```diff
@@ -0,0 +1,609 @@
...
+    {
+      "type": "Feature",
+      "geometry": {
+        "type": "Point",
+        "coordinates": [139.758, 35.675]
+      },
+      "properties": {
+        "place_name": "Tokyo, Japan",
+        "zoom": 12,
+        "description": "The large network of roads inx Tokyo.",
```
Comment (dasulit) [inline mapbox-streets/road-network-gazetteer.json]: Teeny nitpicky: `inx` => `in`

Comment (ClareTrainor) [reply mapbox-streets/road-network-gazetteer.json]: Good catch! tysm

**Pull request merged**